Mittons.Fixtures/Docker/Containers/SftpContainer.cs

```diff
@@ -21,18 +21,24 @@ public class SftpContainer : Container
 
         private readonly IEnumerable<SftpUserAccount> _accounts;
 
+        private readonly int _port;
+
         public SftpContainer(IDockerGateway dockerGateway, Guid instanceId, IEnumerable<Attribute> attributes)
             : base(dockerGateway, instanceId, GetAttributesWithDefaults(attributes))
         {
             _accounts = ExtractSftpUserAccounts(attributes);
+
+            var portBinding = attributes.OfType<PortBinding>().Where(x => x.Protocol == Protocol.Tcp && x.Scheme == "sftp");
```

Comment:
This OfType.Where expression is used multiple times. Any chance to encapsulate and share?

Reply:
I'm not quite sure what you're asking, can you elaborate?